habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -56,8 +67,21 @@ def __init__(
         spatial_size=128,
         make_backbone=None,
         normalize_visual_inputs=False,
+        force_input_size=(256, 256),
     ):
         super().__init__()
+        self.force_input_size = force_input_size
+        observation_space = copy.deepcopy(observation_space)
```

Comment:
Could we make a base class that takes `observation_space` and `force_input_size` and produces `self.observation_space` (then both SimpleCNN and ResNet can share this logic).  That base class could also have a `.forward` method that does the resize.

This could be done in a latter PR of course (there is more unification that could be done than just this).

Reply:
Done